openmdao/utils/tests/test_testing_utils.py

```diff
@@ -0,0 +1,103 @@
+import unittest
+
+from openmdao.utils.testing_utils import use_tempdirs, MissingImports
+
+
+@use_tempdirs
+class TestMissingImports(unittest.TestCase):
```

Comment:
It looks like you are just testing the Missing Imports context manager here...

I think the intent of the Issue was to make sure that openmdao features don't blow up if some optional dependency is not installed, eg. (from the issue description):

```
    with MissingImports('IPython'):
        import openmdao.api as om
```

Just replacing all your imports below with `import openmdao.api` may be a good baseline test...  
Could maybe also try importing some class related to a specific dependency as a further test...